### PR TITLE
Add labels to comments by OP

### LINKED ISSUE #365: Stylize OP name in comments
Similar to how StackOverflow does it, can the OP name be stylized slightly differently when adding a comment to issues and pull requests?

### PR DESCRIPTION
**Pull request by jgierer12**
This adds labels similar to GitHub's existing labels (Owner/Contributor/Collaborator) to comments by the user who submitted the issue/PR.

Timeline comments: Screenshot

With multiple labels: Screenshot with multiple labels

Review comments: Screenshot of review comment

Fixes #365

### CODE REVIEW

File: extension/content.js [hunk 1]
```diff
@@ -277,6 +277,22 @@ function addDiffViewWithoutWhitespaceOption(type) {
...
 }
 
+function addOPLabels(type) {
+	const label = `
+		<span class="timeline-comment-label tooltipped tooltipped-multiline tooltipped-s" aria-label="This user submitted this ${type}.">
+			Original Poster
+		</span>
+	`;
+
+	const comments = $('.js-comment').toArray();
```
Comment (paulmolluzzo) [inline extension/content.js]: `const comments = $('div.js-comment').toArray();`

Without the specificity for a `div` this feature doesn't work when someone has the ability to edit another users comments. (The first item in the array is a `form`.)

**Comment by paulmolluzzo** [comment]
Also, see https://github.com/sindresorhus/refined-github/pull/375#issuecomment-294385505

Thanks for another PR! 🙌

**Comment by paulmolluzzo** [comment]
Any comments on this PR or are we OK to 🚢 ?

**Comment by sindresorhus** [comment]
I don't think we should add "Original Poster" to the initial issue/PR as it's already obvious, but rather only on additional comments by OP.

Merge conflicts needs resolving.

@jgierer12 Can you add it to the readme, like I did here: https://github.com/sindresorhus/refined-github/commit/b695fc43a86965f31f55f740974608d4073dba1c

**Comment by paulmolluzzo** [comment]
> Merge conflicts needs resolving.

@jgierer12 Can you please resolve this? I think we're good otherwise. Thanks!

**Comment by sindresorhus** [comment]
I've also noticed now that if you switch between the "Conversation" tab and "Files" tab in a pull request, it will re-add the OP badge each time you go to "Conversation".

**Comment by jgierer12** [comment]
Could we change `document.addEventListener('DOMContentLoaded', () => {...}` in [this line](https://github.com/sindresorhus/refined-github/blob/master/extension/content.js#L333) to jQuery's `$(document).ready(() => {...})`/`$(() => {...})` instead? Or would that break something?

**Comment by sindresorhus** [comment]
Just add a custom class to the HTML you inject and check for that before injecting. We do that with many of the other features. Could be `.refined-github-op`.

**Comment by paulmolluzzo** [comment]
@sindresorhus anything else?

**Comment by sindresorhus** [comment]
Really good work @jgierer12 :)

**Pull request merged**